.github/workflows/test.yml

```
@@ -8,12 +8,12 @@ permissions:
 jobs:
   test:
     # os is not in the matrix, because otherwise there would be way too many testing instances
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
     strategy:
       fail-fast: false
       matrix:
-        node: [14, 16, 18]
-        mongo: [4.2, 5.0]
+        node: [20, 22]
```

Consider using [this syntax](https://github.com/aheckmann/regexp-clone/blob/main/.github/workflows/ci.yml#L19) which would keep our tested versions in sync with the supported LTS versions. 

```suggestion
        node-version: ['lts/*', 'lts/-1', 'latest']
```

Example of it's use: https://github.com/aheckmann/regexp-clone/actions/runs/17254687292

- 'latest' [mapped to `24.6.0`](https://github.com/aheckmann/regexp-clone/actions/runs/17254687292/job/48964097091)
- 'lts/*' [mapped to `22.18.0`](https://github.com/aheckmann/regexp-clone/actions/runs/17254687292/job/48964097103)
- 'lts/-1' [mapped to `20.19.4`](https://github.com/aheckmann/regexp-clone/actions/runs/17254687292/job/48964097111)

---

Thanks!